compiler/src/iree/compiler/Codegen/Common/unittests/TileSizeSelectionTest.cpp

```diff
@@ -96,8 +96,9 @@ TEST_F(TileSizeSelectionDeathTest, getLevel_out_of_bounds) {
   // 2. Create TilingConfig and verify that the "vector-inner-parallel" tiling
   // level does not exist (it's out of bounds).
   TilingConfig tilingConfig(loweringConfig);
-  ASSERT_DEATH_IF_SUPPORTED({ tilingConfig.getVectorInnerParallelLevel(); },
-                            "Searching for unavailable tiling level");
+  ASSERT_DEATH_IF_SUPPORTED(
+      { tilingConfig.getVectorInnerParallelLevel(); },
+      "Searching for unavailable tiling level");
```

Comment:
@ScottTodd Is the CI clang-format updated? I'm confused that why this is not caught when we check-in the code...

@bjacob Are we able to revert this unrelated change?

Reply:
It looks reverted in the "Files changed" view. CI runs clang-format on changed files. Sometimes different developers / CIs have different versions of clang-format.

Reply:
yeah i had to fix this up in a subsequent commit to get CI clang-format to pass.